src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -139,19 +139,23 @@
           class="mt-6"
           color="red"
         >
-          <p>
-            <v-icon
-              color="red"
-              class="error-icon"
-            >
-              mdi-alert
-            </v-icon>
-            <strong class="pl-2 pt-6">Update directors</strong>
-          </p>
-          <p class="pl-8">
-            A BC Community Contribution Company requires at least three directors.
-            File a director change and then come back and update the business type.
-          </p>
+          <header>
+            <p>
```

Comment:
You don't need the paragraph element. (I think header is equivalent to a div, but we mostly use p for styling and that's probably not needed here.) Test to make sure.

Reply:
tried to remove <p>, but the font of the header is gonna be smaller:
![image](https://github.com/user-attachments/assets/c22082b7-54ee-4232-9bb3-2ec06d380963)

with <p>, same with design:
![image](https://github.com/user-attachments/assets/78470d69-9739-4f04-9cc6-ae588e4da379)

Reply:
Ah, so there's some style that a p has but a header doesn't.  You can add extra styles using the usual helper classes.

Reply:
Thanks. I just updated the style, it looks same as Design now: (updated color to dark-grey, and font-14)
![image](https://github.com/user-attachments/assets/faabc2e3-fc98-4b86-851e-adf547593c21)